lib/manage-prototype-handlers.js

```diff
@@ -361,7 +361,7 @@ async function postTemplatesInstallHandler (req, res) {
 }
 
 function getTemplatesPostInstallHandler (req, res) {
-  const pageName = 'Templates'
+  const pageName = 'Page created from template'
```

Comment:
that's odd it lost my comment - this would would be `Page created` to match the H1